src/models/UserModel.ts

```diff
@@ -1,22 +1,31 @@
-import { Column, Entity, ManyToMany, OneToMany, JoinTable, JoinColumn, PrimaryGeneratedColumn } from 'typeorm';
-
-import { PrivateProfile, Uuid } from '../types';
-import { FeedbackModel } from './FeedbackModel';
-import { PostModel } from './PostModel';
-import { RequestModel } from './RequestModel';
-import { UserSessionModel } from './UserSessionModel';
-import { UserReviewModel } from './UserReviewModel'
-
-@Entity('User')
+import {
+  Column,
+  Entity,
+  ManyToMany,
+  OneToMany,
+  JoinTable,
+  JoinColumn,
+  PrimaryGeneratedColumn,
+} from "typeorm";
+
+import { PrivateProfile, Uuid } from "../types";
+import { FeedbackModel } from "./FeedbackModel";
+import { PostModel } from "./PostModel";
+import { RequestModel } from "./RequestModel";
+import { UserSessionModel } from "./UserSessionModel";
+import { UserReviewModel } from "./UserReviewModel";
+import { ReportModel } from "./ReportModel";
+import { MessageModel } from "./MessageModel";
+
+@Entity("User")
 export class UserModel {
-
-  @PrimaryGeneratedColumn('uuid')
+  @PrimaryGeneratedColumn("uuid")
   id: Uuid;
 
   @Column({ unique: true })
   username: string;
 
-  @Column({ unique: true })
+  @Column({ unique: true, nullable: true })
```

Comment:
I'd make sure that frontend knows to make the necessary change for this to work as intended, i.e., they should be sending over a `null` netid field when authenticating a user without a Cornell email (instead of an empty string, which is what they were doing before)